scripts/l3bgp_demo.sh

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
This is just a description of the l3bgp-demo setup. BGP documentation is already present in contiv doc/tutorial.